aperture.go

```diff
@@ -209,6 +209,13 @@ func getConfig(configFile string) (*config, error) {
 	if cfg.ListenAddr == "" {
 		return nil, fmt.Errorf("missing listen address for server")
 	}
+
+	for _, service := range cfg.Services {
+		if service.Price == 0 {
```

Comment:
I think we should define a default value of 1 satoshi and use that if no price is defined on a service. See comment further below. But we should check if the price is negative and then error out (because the field is defined as `int64`).